thoth/storages/database_backup.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# thoth-storages
+# Copyright(C) 2019 Harshad Reddy Nalla
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Adapter for storing database backups."""
+
+from .result_base import ResultStorageBase
+
+
+class DatabaseBackupStore(ResultStorageBase):
+    """Adapter for storing database backups."""
+
+    RESULT_TYPE = "database-backup"
```

Comment:
What about:
```
    RESULT_TYPE = "graph-backup"
```

Reply:
Also, let's rename `DatabaseBackupStore` to `GraphBackupStore` to be consistent with "graph" naming in sources.